### DISCUSSION TITLE
how to perform gpu passthrough to be able to play directx games

### DISCUSSION THREAD

**Discussion by firefinchdev** [utmapp/UTM#6658] [UTM Help (macOS)]
**Describe the issue**  
Hi,
I want to play a game (it is very **lightweight** for CPU) on UTM Windows XP x86. I have installed DirectX but the game reports `d3derr_notavailable` on starting. After searching, it seems like I somehow need to bypass gpu, or need a driver compatible with virtio-vga-gl.
Sorry if i misunderstood something.
Any way to be able to do so?
Thanks in advance.

**Configuration**  
* UTM Version: 100
* macOS Version: Sonoma 14.5
* Mac Chip (Intel, M1, ...): M2 Pro

**Comment by firefinchdev** [comment]
To add to the context above, maybe this post can help a bit: https://forum.mattkc.com/viewtopic.php?t=206
But i dont have any idea how to apply this to UTM

**Comment by osy** [comment]
UTM currently doesn't support GPU for windows
